### DISCUSSION TITLE
Fastest route for car Navigation

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#15535] [Q&A]
How to force OSMAnd to use the FASTEST route with car-profile. In my case OSMAnd calculate the SHORTEST Track. When I add a waypoints, the route is a little bit longer, but also some minutes faster.
How can I tell OSMAnd to calculate the FASTEST route? In Bike-Profile there is an Option for that. In car not...

**Comment by ThomasMeister123** [comment]
No one any idea how to get the FASTES Route in Car-Profile? (with or without modifying the routing.xml)

**Comment by pebogufi** [comment]
Fastest route is default at car navigation, it works if you don't switch it of.
In navigation options of car you find "fuel efficient route"/"Kraftstoff sparende Route" which is sometimes a bit shorter. The default (off) uses higher priority roads, which should be faster. If ON it uses a more direct routing and also smaler (= less priority) roads. This _might_ be shorter.

**Reply by droidguyg1** [reply]
Unfortunately, it doesn't behave like this.  See the attached screenshot. With fuel-efficient turned off (widget is grey, the long northbound leg of the route follows a residential slow street without even benefiting from it distance-wise.  In this case, the fuel efficient route (widget is blue) is already a bit better.
![Screenshot_2023-07-22-08-29-01](https://github.com/osmandapp/OsmAnd/assets/9326711/093b49be-c5f8-411f-99a7-a84b0e6f075f)
![Screenshot_2023-07-22-08-29-26](https://github.com/osmandapp/OsmAnd/assets/9326711/2ddc80e8-6ffe-4640-a1e7-06c0360a48ce)

**Comment by vshcherb** [comment]
Fastest could mean how good penalties are synchronized in routing.xml and in real life, https://github.com/osmandapp/OsmAnd-resources/blob/master/routing/routing.xml. 

You can experiment with cost on website with OsmAnd routing - https://test.osmand.net/map?start=48.610670,9.299240&end=48.595230,9.288254&type=osmand&profile=car#14/48.6044/9.3232
